src/main/java/org/jboss/config/LotteryConfigProducer.java

```diff
@@ -0,0 +1,34 @@
+package org.jboss.config;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+
+@ApplicationScoped
+public class LotteryConfigProducer {
+
+    private LotteryConfig lotteryConfig;
+    @Inject
+    ObjectMapper objectMapper;
+
+    @Inject
+    JiraLotteryAppConfig jiraLotteryAppConfig;
+
+    public LotteryConfig getLotteryConfig() {
```

Comment:
That was the initial intention, however due to mocking in tests I had to create a producer bean and furthermore, I had to create this getter instead of using `@PostConstruct`

Reply:
I don't remember the context here. but it seems to be resolved. @The-Huginn resolve this comment if it is.